Package.swift

```diff
@@ -40,15 +41,21 @@ let package = Package(
                 .product(name: "RuntimeAssertions", package: "XCTRuntimeAssertions"),
                 .product(name: "OrderedCollections", package: "swift-collections")
             ],
-            swiftSettings: [.enableUpcomingFeature("ExistentialAny")],
+            swiftSettings: [
+                .enableUpcomingFeature("ExistentialAny"),
+                .enableUpcomingFeature("InternalImportsByDefault"), .unsafeFlags(["-enable-testing"])
```

Comment:
I think we can't work with any unsafe flags for exposed packages: https://developer.apple.com/documentation/packagedescription/swiftsetting/unsafeflags(_:_:). We would need to remove them before merging the PR.